scripts/update-version.js

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env node
+
+/*
+
+* Get current npm version.
+* Determine what type of version bump: major, minor, patch
+* Update the files:
+  * package.json
+  * android/gradle.properties
+  * react-native-maps.podspec
+  * react-native-google-maps.podspec
+* Make Git commit.
+* Create Git tag.
+
+*/
+
+// TODO: update eslint-plugin-import so we can configure
+// `import/no-extraneous-dependencies` to allow devDependencies in `scripts/`.
+
+const { exec } = require('child_process');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const commander = require('commander');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const dedent = require('dedent');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const semver = require('semver');
+const pkg = require('../package.json');
+
+const versionTypes = ['major', 'minor', 'patch'];
+const filesToUpdate = [
+  'package.json',
+  'android/gradle.properties',
+  'react-native-maps.podspec',
+  'react-native-google-maps.podspec',
```

Comment:
are all three of these always in sync? What happens when there's a bugfix that only applies to one of them?